highlight_view.py

```diff
@@ -989,4 +1029,382 @@ def sort_by_type(error_type):
             heading=heading,
             messages='<br />'.join(filled_templates)
         )
-    return all_msgs
+    return all_msgs, quick_actions
+
+
+def escape_text(text):
+    # type: (str) -> str
+    return html.escape(text, quote=False).replace(' ', '&nbsp;')
+
+
+from contextlib import contextmanager
+if MYPY:
+    from typing import Iterator, NamedTuple
+    Action = NamedTuple("Action", [("description", str), ("fn", Callable)])
+    TextRange = NamedTuple("TextRange", [("text", str), ("range", sublime.Region)])
+else:
+    from collections import namedtuple
+    Action = namedtuple("Action", "description fn")
+    TextRange = namedtuple("TextRange", "text range")
+
+
+class sl_fix_by_ignoring(sublime_plugin.TextCommand):
+    def run(self, edit):
+        view = self.view
+        window = view.window()
+        assert window
+        sel = [s for s in view.sel()]
+        if len(sel) > 1:
+            window.status_message("Only one cursor please.")
```

Comment:
How do you feel about moving these checks into an `is_enabled()` check? This would remove the command from the list. In the context menu it would be greyed out. That doesn't explain "why", but the feedback in the status bar is also very easy to miss.
Of course, eventually we might be able to support multiple cursors as well (ie. "simply" loop through all cursor positions and handle them one by one)?

Reply:
The `status_message` calls are just short hacks.  For example in the case of `status_message("No errors here.")` I usually show the quick panel with an error message and a `noop` callback so that both `ESC` and `Enter` just closes the panel. 

Here, you're correct I wasn't sure if we can support multiple cursors. If not, I would prefer `is_enabled()`. At the moment I don't think multiple cursors or even multiple selections lead to a good experience.

Reply:
> At the moment I don't think multiple cursors or even multiple selections lead to a good experience.

It probably isn't. I usually do prefer trying to support multiple cursors even if it probably doesn't ever happen in the wild, it's just such a corner stone of the Sublime Text experience that everything you can do with one also works with n cursors.